static/app/components/events/interfaces/crashContent/exception/content.spec.tsx

```diff
@@ -148,12 +148,12 @@ describe('Exception Content', function () {
       })
     ).toHaveAttribute(
       'href',
-      '/settings/org-slug/projects/project-slug/security-and-privacy/#advanced-data-scrubbing'
+      '/settings/org-slug/projects/project-slug/security-and-privacy/advanced-data-scrubbing/0/'
     );
 
     expect(screen.getByRole('link', {name: 'project-slug'})).toHaveAttribute(
       'href',
-      '/settings/org-slug/projects/project-slug/security-and-privacy/#advanced-data-scrubbing'
+      '/settings/org-slug/projects/project-slug/advanced-data-scrubbing/'
```

Comment:
This seems wrong?

https://github.com/getsentry/sentry/pull/38734/files?diff=split&w=1#diff-5154b281a07ffcda3774e4f467493dc58d9d725e94223f0d9641e82989819952R416

Reply:
oh yes, good catch this should be `/security-and-privacy/advanced-data-scrubbing/`..thank you